rotkehlchen/accounting/structures/base.py

```diff
@@ -325,7 +325,7 @@ def __init__(
         )
 
     def __repr__(self) -> str:
-        return f'HistoryEvent({",".join(self._history_base_entry_repr_fields())})'
+        return f'HistoryEvent({", ".join(self._history_base_entry_repr_fields())})'
```

Comment:
is because when you show an evm using the repr the fields are compated in the form `field_value,self.other_field=` and is hard to read. This spacing shows them more like the dataclass does it